index.php

```diff
@@ -63,7 +63,7 @@
                         </p>
                         <div class="text-right d-flex justify-content-end">
                             <!-- * href内を変更する -->
-                            <a href="edit.php" class="btn text-success">EDIT</a>
+                            <a href="edit.php?id=<? h($task['id']) : ?>" class="btn text-success">EDIT</a>
```

Comment:
最後はコロンじゃなくてセミコロンですね！
```suggestion
                            <a href="edit.php?id=<? h($task['id']) ; ?>" class="btn text-success">EDIT</a>
```